build-tools/packages/build-cli/src/library/repoPolicyCheck/fluidBuildTasks.ts

```diff
@@ -249,20 +171,28 @@ function findTscScripts(json: Readonly<PackageJson>, project: string): string[]
 	addIfDefined(findScript(json, `tsc --project ${project}`));
 	addIfDefined(findFluidTscScript(json, project));
 	addIfDefined(findTscMultiScript(json, project));
-	return tscScripts.length > 0 ? tscScripts : undefined;
+	if (tscScripts.length === 1) {
+		return tscScripts[0];
+	}
+	if (tscScripts.length === 0) {
+		return undefined;
+	}
+	throw new Error(`${project} used in scripts ${tscScripts.join(", ")}`);
```

Comment:
So any potential incorrect usage where project or tscScripts are empty/blank produces a message that's easier to interpret.
```suggestion
	throw new Error(`'${project}' used in scripts '${tscScripts.join(", ")}'`);
```